src/protocol/LSP.CodeAction.pas

```diff
@@ -39,7 +39,7 @@ interface
     The set of kinds is open and client needs to announce the kinds it supports to the server during
     initialization. }
 
-  TCodeActionKind = record
+  TCodeActionKind = Class
```

Comment:
Why was this made a class?

Reply:
Because you don't need the value field in a class. The compiler complained on the value field.

The intent clearly was to create a namespace, and then a class is better suited.

Personally I would do away with the record/class alltogether, it serves IMO no purpose.
Typical Java programmer stuff, I suppose.